### PR TITLE
chore(set up test functionality): Setup test functionality with jest …

### PR DESCRIPTION
**Pull request by koiic**
#### What does this PR do?
 - setup test functionality with jest and enzyme
 -  Write a dummy example test

#### Description of Task to be completed?
- add Jest configuration
- add enzyme and react dependencies
- setup a dummy test in the  ```__tests__``` folder

#### How should this be manually tested?
- Install dependencies ```yarn add```
- start the test by running ```yarn test``` or ```yarn test:watch```
the test should return and show you the result in the terminal

#### What are the relevant pivotal tracker stories?
[#160451550](https://www.pivotaltracker.com/story/show/160451550, "setup test functionality")

### CODE REVIEW

File: src/components/__tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+configure({ adapter: new Adapter() });
+
+
+//SnapShot Test
+describe('App', () => {
+  it('should render correctly in "debug" mode', () => {
+    const component = shallow(<Home debug />);
+    expect(component).toMatchSnapshot();
+  });
+
```
Comment (KaiserPhemi) [inline src/components/__tests__/app.test.js:19]: #### Nitpick
Remove this blank line. One blank line is needed to separate test cases

File: src/components/__tests__/app.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  it('should render banner text correctly with given strings', () => {
+    const strings = ['one', 'two'];
+    const component = shallow(<Home list={strings} />);
+    expect(component).toMatchSnapshot();
+  });
+});
```
Comment (KaiserPhemi) [inline src/components/__tests__/app.test.js]: #### Nitpick
Add a new line at the end of the file

File: src/components/__tests__/app.test.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```
Comment (KaiserPhemi) [inline src/components/__tests__/app.test.js:2]: #### Nitpick
Add comment to this
i.e. 
```js
// react libraries
import React from 'react';
```

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
@koiic 
Implement feedback

File: src/components/__tests__/app.test.js (hunk 3 above)
Comment (koiic) [reply src/components/__tests__/app.test.js:2]: Done

File: src/components/__tests__/app.test.js (hunk 1 above)
Comment (koiic) [reply src/components/__tests__/app.test.js:19]: Done

File: src/components/__tests__/app.test.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+// third-party libraries
+import { shallow, configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+// modules importation
+import Home from '../Home';
+
+configure({ adapter: new Adapter() });
+
+
```
Comment (andela-phemmy) [inline src/components/__tests__/app.test.js]: NIce work @koiic , you should remove this blank line to make the code more readable

Comment (koiic) [reply src/components/__tests__/app.test.js]: Thanks...I will work on it

**Review by andela-phemmy — APPROVED** [review]

**Review by e0works — APPROVED** [review]
It looks good to me.

**Review by KaiserPhemi — APPROVED** [review]

**Pull request merged**